demo-terraform/aws/serverless-app/s3.tf

```diff
@@ -1,16 +1,38 @@
 
 ## Create random id
 resource "random_string" "uniq" {
-    length = 32
-    special = false
-    lower = true
-    upper = false
+  length  = 32
+  special = false
+  lower   = true
+  upper   = false
 }
 
 ## Create bucket
 resource "aws_s3_bucket" "website" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure bucket policy is enforced with least privileges for all AWS S3 buckets**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
Make sure to note Amazon S3 bucket access control lists (ACLs) that provide read, write, or full-access to 'Everyone'. It is recommended to provide access to S3 buckets to only authenticated users instead.

source: accurics-